### PR TITLE
added route to extract personal data from the auth service

### PR DESCRIPTION
**Pull request by jlotery**
Goal of the PR was to extract personal data from the auth service belonging to a user.

### CODE REVIEW

**Review by gabila-r — APPROVED** [review]
LGTM

File: services/auth/lib/handlers.js [hunk 1]
```diff
@@ -392,6 +392,39 @@ function addGroupsToUser(req, res) {
...
+    req.db.collection('__users__').findOne(
+      { _id: userId },
+      {
+        projection: {
+          email: 1,
+          displayname: 1,
+          picture: 1,
+          identities: {
+            local: { id: 1, username: 1 },
+            google: { id: 1, sub: 1, name: 1, given_name: 1, familly_name: 1, picture: 1, email: 1 },
```
Comment (rombat) [inline services/auth/lib/handlers.js:413]: maybe the `locale` prop would be useful too

**Review by rombat — APPROVED** [review]

File: services/auth/lib/handlers.js (hunk 1 above)
Comment (jlotery) [reply services/auth/lib/handlers.js:413]: can you confirm the prop name, do you mean preferedLocale? Thanks :)

Comment (rombat) [reply services/auth/lib/handlers.js:413]: google identity can have a `locale` prop. I think it can be useful

Comment (rombat) [reply services/auth/lib/handlers.js:413]: but you can save preferredLocale and preferredLanguage too

**Pull request merged**